visualization/app/codeCharta/util/customConfigHelper.ts

```diff
@@ -227,19 +225,8 @@ export class CustomConfigHelper {
 		const customConfig = this.getCustomConfigSettings(configId)
 		CustomConfigHelper.transformLegacyCameraSettingsOfCustomConfig(customConfig)
 
-		// TODO: Setting state from loaded CustomConfig not working at the moment
-		//  due to issues of the event architecture.
-
-		// TODO: Check if state properties differ
-		// Create new partial State (updates) for changed values only
 		store.dispatch(setState(customConfig.stateSettings))
 
-		// Should we fire another event "ResettingStateFinishedEvent"
-		// We could add a listener then to reset the camera
-
-		store.dispatch(setColorRange(customConfig.stateSettings.dynamicSettings.colorRange))
```

Comment:
this is actual needed. Otherwise wie set the state, which sets blacklist to [], which leads to recalculation of selectedColorMetricDataSelector and therefore resetColorRange.effect is triggered. 

I wonder if it is really expected to reset color range when excluding a node? I suggest to remove `selectedColorMetricDataSelector` from resetColorRange.effect.

Reply:
This should not be the case. Excluding a file/folder should not reset the color range.

Reply:
I will update accordingly on Monday

Reply:
done